apps/condo/domains/miniapp/constants.js

```diff
@@ -74,6 +74,9 @@ const B2C_APP_COLOR_SCHEMA_TYPE_NAME = 'AppColorSchemaField'
 const B2C_APP_COLOR_SCHEMA_INPUT_NAME = 'AppColorSchemaFieldInput'
 const GLOBAL_APP_NO_APP_URL_ERROR = '[app:noAppUrl] If the app is global, it must have appUrl field'
 const NON_GLOBAL_APP_WITH_FEATURES_ERROR = '[app:non-global:features] Non global apps cannot specify features'
+const USER_NOT_FOUND_ERROR = 'USER_NOT_FOUND'
```

Comment:
check, probably we already have this error in others mutations?

Reply:
Yes, but not in this domain 
Importing errors from other domains is probably a bad practice